File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        String odd = "";
        String even = "";
        
        for(int num : num_list){
            odd  += (num % 2 == 1) ? num : "";
            even += (num % 2 == 0) ? num : "";
        }
        
        return Integer.parseInt(odd) + Integer.parseInt(even);
    }
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    odd = ""
    even = ""
    
    for n in range(len(num_list)):
        n = num_list[n]
        if n%2==0:
            even += str(n)
        else:
            odd += str(n)
    return int(even)+int(odd)